webdriver-spec.html

```diff
@@ -1314,6 +1320,12 @@
     return an <a>error</a> with code <a>timeout</a>.
   </ol>
 
+ <li><p>Set the <a>current browsing context</a> to the
+  <a>current top-level browsing context</a> as if <a>Switch to Frame</a>
+  command was executed with no specified
+  <a href="https://html.spec.whatwg.org/#child-browsing-context">
+  child browsing context</a>.
```

Comment:
Agreed, 

" as if <a>Switch to Frame</a>
 command was executed with no specified
 <a href="https://html.spec.whatwg.org/#child-browsing-context">
 child browsing context</a>"

can be removed completely. I can update this PR when the other comment in this PR is resolved.